packages/core/src/sass/app/modules/welcome.scss

```diff
@@ -100,7 +100,8 @@
 
         &__option {
             padding: 0 0.2rem;
-            border-right: 1px solid var(--general-section-1);
+            color: #333333;
```

Comment:
You will need to use a theme color from theme.scss for this such as `--icon-grey-background` to handle dark theme as well.